content/posts/2021-11-18-relicensing-update.md

```diff
@@ -0,0 +1,328 @@
+    process.
+
+    The LLVM exception enables the run-time libraries to be covered by the exact
+    same single license as the rest of the code base.  
+
+# Getting all code covered by the new license
+
+After a decision was made of what the new license should be, we started working
+on having all code covered by it.
+            
+As a first step, we made sure that all new contributions were covered by the new
+license. This happened after the 8.0 release branch was created. The 100k
+commits since are covered by the new license.
+            
+The remaining task is to also get the earlier contributions covered by the new
+license. This consists of about 300k commits totaling about 32 million lines of
+contributed code.
+
+<img>
+<svg width=700 height=400 style="mix-blend-mode: difference;">
+  <rect x="0" y="200" width="350" height="50" style="fill:rgb(0, 183, 255);stroke-width:0" />
+  <rect x="350" y="200" width="350" height="50" style="fill:rgb(72, 255, 0);stroke-width:0" />
+  <rect x="347" y="200" width="6" height="50" style="fill:gray;stroke-width:0"/>
+  <text x="350" y="275" style="fill:rgb(255, 255, 255);font-size: 20px;"
+  text-anchor="middle">2019/01/19</text>
+  <text x="700" y="275" style="fill:rgb(255, 255, 255);font-size: 20px;"
+  text-anchor="end">Today</text>
+  <text x="0" y="275" style="fill:rgb(255, 255, 255);font-size: 20px;"
+  text-anchor="start">2001/06/06</text>
+
+  <g class="fragment">
+    <circle cx="390" cy="47" r="5" stroke-width="0" fill="gray"/>
+    <line x1="390" y1="47" x2="350" y2="200" style="stroke:gray;stroke-width:2" />
+    <circle cx="350" cy="200" r="5" stroke-width="0" fill="gray"/>
+    <foreignObject x="400" y="20" width="300" height="200">
+    <div xmlns="http://www.w3.org/1999/xhtml"
+    style="font-size:20px;text-align:left">
+    New license installed after<br/>
+    8.0 release branched.
+    </div>
+  </foreignObject>
+  </g>
+
+  <g class="fragment">
+    <circle cx="420" cy="290" r="5" stroke-width="0" fill="gray"/>
+    <line x1="420" y1="290" x2="420" y2="225" style="stroke:gray;stroke-width:2" />
+    <circle cx="420" cy="225" r="5" stroke-width="0" fill="gray"/>
+    <foreignObject x="400" y="300" width="300" height="200">
+    <div xmlns="http://www.w3.org/1999/xhtml"
+    style="font-size:20px;text-align:left">
+    All code since is covered.
+    (~100k commits so far)
+    </div>
+  </foreignObject>
+  </g>
+
+  <g class="fragment">
+    <circle cx="40" cy="47" r="5" stroke-width="0" fill="gray"/>
+    <line x1="40" y1="47" x2="40" y2="225" style="stroke:gray;stroke-width:2" />
+    <circle cx="40" cy="225" r="5" stroke-width="0" fill="gray"/>
+    <foreignObject x="50" y="20" width="300" height="200">
+    <div xmlns="http://www.w3.org/1999/xhtml"
+    style="font-size:20px;text-align:left">
+    Most code before needs agreement from copyright owners.
+    (~300k commits; ~32M LOC)
+    </div>
+  </foreignObject>
+  </g>
+</svg>
+</img>
```

Comment:
Similarly, this isn't working for me in the preview.  It might be better to go with images.

Reply:
Will do.